app/helpers/application_helper/button/transform_vm_button.rb

```diff
@@ -0,0 +1,12 @@
+class ApplicationHelper::Button::TransformVmButton < ApplicationHelper::Button::Basic
+  needs :@record
+
+  def visible?
+    @record.vendor == "vmware"
+  end
+
+  def disabled?
+    # Is there a provider that supports import?
+    EmsInfra.all.select(&:validate_import_vm).empty?
+  end
```

Comment:
I guess users have less than 100 providers most of the time ;-)

Unlike VMs or containers where there can be 10 or 100 of 1000.